release/tasks/disk-uploader/README.md

```diff
@@ -37,6 +52,10 @@ spec:
         value: example-dv
     -   name: IMAGE_DESTINATION
         value: quay.io/kubevirt/example-vm-exported:latest
+    -   name: PUSH_TIMEOUT
```

Comment:
If the  PUSH_TIMEOUT would not be included in the example, the example would be more simple.